signac/contrib/import_export.py

```diff
@@ -117,6 +205,19 @@ def path_function(job):
         paths = _make_schema_based_path_function(jobs=jobs, exclude_keys=exclude_keys)
 
         def path_function(job):
+            """Format a path for a given job.
```

Comment:
```suggestion
            """Format a path based on a given string and schema.
```
There might be a better way to describe this, but the current doc-string is very generic. If we go with that generic description, then we should keep it generic for the other path functions as well.